sdk/storage/storage-file-datalake/test/utils/testutils.common.ts

```diff
@@ -99,7 +99,7 @@ export class SimpleTokenCredential implements TokenCredential {
 }
 
 export function isBrowser(): boolean {
-  return typeof window !== "undefined";
+  return typeof self !== "undefined";
```

Comment:
Can self also defined when it's not a browser environment?  
I searched around, seems self can exist no matter whether there's a windows context:

> By using self, you can refer to the global scope in a way that will work not only in a window context (self will resolve to window.self) but also in a worker context (self will then resolve to WorkerGlobalScope.self).